### ISSUE TITLE
Netflix IP Issue

### ISSUE THREAD

**Issue by exedox** [brave/brave-browser#13357]
## Description 
netflix.com’s server IP address could not be found. error pops up on Brave webbrowser.

## Steps to Reproduce
Inputting netflix webaddress
## Actual result:
netflix.com’s server IP address could not be found.

## Expected result:
Netflix website

## Reproduces how often: 
All the time, on two separate devices. Android is fine.
Other browsers can access Netflix

## Brave version (brave://version info)

Brave | 1.18.75 Chromium: 87.0.4280.101 (Official Build) (64-bit)
-- | --
Revision | 9407c80213cda69c2b7abcb4fa8e3f74488f4956-refs/branch-heads/4280@{#1807}
OS | Windows 10 OS Version 1909 (Build 18363.1256)

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by rebron** [comment]
@exedox You're saying you're not able to visit `https://www.netflix.com` and it works in other browsers on Windows on your same machine?

**Comment by exedox** [comment]
Yes it works on chrome and edge on my main computer.  When I attempt to go to netflix on brave, it gives the error I listed. I also tested this on other devices, only windows 10 brave has the issue. Android is ok, but brave on my windows 10 laptop doesn't work. @rebron 

I also attempted to create another profile to see if it works, and it doesnt.

**Comment by kjozwiak** [comment]
@exedox mind taking a screenshot of the error message you're getting when loading netflix.com?

**Comment by exedox** [comment]
no problem @kjozwiak 
![image](https://user-images.githubusercontent.com/55308770/103743908-b0dcd400-4fb1-11eb-9fb9-0101cbda15bd.png)

**Comment by ryanbr** [comment]
Hey, @exedox does the same error message show in Chrome (or any browser).

Also if you're using a VPN, change to a different location (or turn off)

**Comment by exedox** [comment]
hi @ryanbr Chrome and Edge can access netflix homepage. brave does not.

**Comment by ryanbr** [comment]
hmm okay,

Does netflix work in private window mode, no extensions enabled? 
Can you open `cmd` in windows and `ping www.netflix.com`, does it timeout or respond?

**Comment by exedox** [comment]
@ryanbr i asssume private window is guest window? In that case, no it doesn't work. 

In cmd it times out. that's kinda odd considering it works ok on the chrome browser.

**Comment by ryanbr** [comment]
Possibly Chrome is showing a cached version? If you're using a VPN these will affect these results
From NZ->USA Netflix

```
ping www.netflix.com

Pinging www.netflix.com [89.187.181.38] with 32 bytes of data:
Reply from 89.187.181.38: bytes=32 time=215ms TTL=49
Reply from 89.187.181.38: bytes=32 time=214ms TTL=49
Reply from 89.187.181.38: bytes=32 time=213ms TTL=49
Reply from 89.187.181.38: bytes=32 time=214ms TTL=49

Ping statistics for 89.187.181.38:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 213ms, Maximum = 215ms, Average = 214ms
```

**Comment by exedox** [comment]
@ryanbr thanks ryan, so chatting with everyone got me thinking maybe it's my router's fault so I reset it.  That seemed to resolve the issue, I also added netflix.com to my pi-hole whitelist.  Ping looks different for mine, and I'm not sure why

```Pinging dualstack.apiproxy-website-nlb-prod-1-bcf28d21f4bbcf2c.elb.us-west-2.amazonaws.com [52.38.7.83] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 52.38.7.83:
    Packets: Sent = 4, Received = 0, Lost = 4 (100% loss),```

However now this problem has been fixed.  I just don't know what could have caused the issue really because if pi-hole was working properlyl then all browsers would block netflix (didn't prior).  So I"m going to assume it was due to my router.
